cmake/onnx.cmake

```diff
@@ -0,0 +1,34 @@
+PROTOBUF_GENERATE_CPP(PROTO_SRCS PROTO_HDRS
+    ${ONNXIR_ROOT}/onnx/onnx.proto
+    ${ONNXIR_ROOT}/onnx/onnx-operators.proto
+)
+
+file(GLOB_RECURSE onnx_src
+    "${ONNXIR_ROOT}/onnx/*.h"
+    "${ONNXIR_ROOT}/onnx/*.cc"
+)
+
+file(GLOB_RECURSE onnx_exclude_src
+    "${ONNXIR_ROOT}/onnx/onnx.pb.h"
+    "${ONNXIR_ROOT}/onnx/onnx.pb.cc"
+    "${ONNXIR_ROOT}/onnx/onnx-operators.pb.h"
+    "${ONNXIR_ROOT}/onnx/onnx-operators.pb.cc"
+)
+list(LENGTH onnx_exclude_src list_len)
+if (list_len GREATER 1)
+    list(REMOVE_ITEM onnx_src ${onnx_exclude_src})
```

Comment:
Do you mind adding a piece of comment about the need for the list removal?

Reply:
this is temporary thing, since setup.py is not changed to use cmake for now. The proto header/code files will also be generated by setup.py (pip install -e ./), so, I'm adding the temp logic here to remove them if they're generated. It should be removed later.